02_Util/src/util/validator.ts

```
@@ -302,10 +385,18 @@ export function validateIdToken(
           'Central tokens must contain only letters, numbers, and characters: * - _ = : + | @ .',
       };
 
-    default:
+    case OCPP2_0_1.IdTokenEnumType.eMAID:
+      if (validateEMAIDIdToken(idToken)) {
```

I think `validateEMAIDIdToken` will return list of errors and list of errors is initialized to `[]` and if I try:
```
if ([]) { console.log('true'); }
```
It logs true, so I think it may be unintentionally returning `{ isValid: true}`? 

I was thinking, maybe we can return the errors in the `errorMessage` below?